sharding-core/sharding-core-execute/src/main/java/org/apache/shardingsphere/core/execute/metadata/TableMetaDataLoader.java

```diff
@@ -165,8 +174,9 @@ private boolean isTableExist(final Connection connection, final String catalog,
                 String columnName = resultSet.getString(COLUMN_NAME);
                 String columnType = resultSet.getString(TYPE_NAME);
                 boolean isPrimaryKey = primaryKeys.contains(columnName);
-                boolean isNotNull = isPrimaryKey || !resultSet.getBoolean(IS_NULLABLE);
-                boolean isAutoIncrement = resultSet.getBoolean(IS_AUTOINCREMENT);
+                boolean isNotNull = isPrimaryKey || !resultSet.getString(IS_NULLABLE).equalsIgnoreCase("YES");
+                String autoIncrement = resultSet.getString(IS_AUTOINCREMENT);
+                boolean isAutoIncrement = autoIncrement.equalsIgnoreCase("YES");
```

Comment:
Hi @beijing-penguin ,

Why `IS_NULLABLE ` and `IS_AUTOINCREMENT ` use `getString()` method?

Does The `getBoolean ` method get mistake value in some database?

Reply:
because  IS_NULLABLE and IS_AUTOINCREMENT return type  is String "YES",not boolean..